test/parallel/test-readline-position.js

```diff
@@ -14,16 +14,23 @@ const ctrlU = { ctrl: true, name: 'u' };
     prompt: ''
   });
 
-  for (const [cursor, string] of [
+  const tests = [
```

Comment:
This is necessary to fix building (and testing) on ICU, since the JS implementation can only distinguish between wide/narrow.